src/main/java/com/aws/greengrass/security/exceptions/ServiceUnavailableException.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.security.exceptions;
+
+public class ServiceUnavailableException extends Exception {
+
+    static final long serialVersionUID = -3387516993124229948L;
```

Comment:
nit: Shouldnt serialVersionUID be unique across classes?

Reply:
I think `serialization id` is intended for distinguishing different version of class. So It doesn't need to be unique across classes.

Reply:
ahh ok.